.gitignore

```diff
@@ -12,7 +12,12 @@ target
 .idea
 *.iml
 *.log
-/test/output
+/test/screenshots/diff/**/*
+!/test/screenshots/diff/.empty
```

Comment:
I would suggest using `mkdirp.sync(dirname(path))` before saving, rather than this. It's more future proof and less messy that way.